drivers/hyperv/hyperv.go

```diff
@@ -398,6 +410,10 @@ func (d *Driver) Kill() error {
 }
 
 func (d *Driver) GetIP() (string, error) {
```

Comment:
over time we could get rid of these functions

Reply:
yes!